deliveries/multisignature_management_tool_milestone_2.md

```diff
@@ -0,0 +1,18 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** Please, provide a link to the merged contract (the `.md` file in the [applications](https://github.com/w3f/Grants-Program/blob/master/applications/multisignature_management_tool.md) directory). 
+* **Milestone Number:** 2
+
+
+| Number | Deliverable | Link | Notes |
+| --- | --- | --- | --- |
+| 0. | Support more kinds of Extrinsic | [pr](https://github.com/itering/subscan-multisig-react/commit/147ce7e64e403b157d43389326d12a92da95fb64) | 
```

Comment:
For what extrinsics has support been added with this PR exactly? The application mentions "Extrinsics such as staking, Democracy".

Reply:
We imported and modified react-components from polkadot apps to support all extrinsics exists in listed networks.